app/models/deepl_client.py

```
@@ -0,0 +1,82 @@
+            >>> result = service.translate("Hello", "JA")
+            >>> print(result)
+            'こんにちは'
+        """
+        # 空文字列チェック
+        if not text or not text.strip():
+            return ""
+
+        # ここに書かないと循環インポートになる
+        from app.main import flask_cache
+
+        question_hash = hashlib.md5(text.encode()).hexdigest()
```

Ruff (0.14.8)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)